components/style/properties/properties.mako.rs

```diff
@@ -1676,6 +1675,7 @@ pub fn get_writing_mode(inheritedbox_style: &style_structs::InheritedBox) -> Wri
         computed_values::text_orientation::T::mixed => {},
         computed_values::text_orientation::T::upright => {},
     % endif
+        computed_values::text_orientation::T::sideways_right => {},
```

Comment:
If `sideways-right` is an alias of `sideways`, why do they have different effect here?

Gecko drops `sideways-right` value (parses it as `sideways`), which I guess Servo can do as well. Is there any reason to keep Servo's `sideways-right` separate from `sideways`?